RecoLocalCalo/HcalRecAlgos/src/MahiFit.cc

```diff
@@ -548,14 +546,46 @@ void MahiFit::phase1Debug(const HBHEChannelInfo& channelData, MahiDebugInfo& mdi
     } else if (nnlsWork_.bxs.coeff(iBX) == pedestalBX_) {
       mdi.pedEnergy = nnlsWork_.ampVec.coeff(iBX);
     } else if (nnlsWork_.bxs.coeff(iBX) == -1) {
-      mdi.pEnergy = nnlsWork_.ampVec.coeff(iBX);
+      mdi.p1Energy = 0;
+      mdi.p1Energy = nnlsWork_.ampVec.coeff(iBX);
```

Comment:
changes here seem to go beyond the PR description, which mentions only "Bug fix in initializing nP variable".
Please check and modify the PR description appropriately.

Reply:
ah, nm, this is in the ```phase1Debug``` method.